contracts/ExitHandler.sol

```diff
@@ -102,12 +103,21 @@ contract ExitHandler is DepositHandler {
     TxLib.Output memory out = exitingTx.outs[_outputIndex];
 
     bytes32 utxoId = bytes32(uint256(_outputIndex) << 120 | uint120(uint256(txHash)));
-    require(out.owner == msg.sender, "Only UTXO owner can start exit");
+    uint256 priority;
+    if (msg.sender != out.owner) {
+      // or caller code hashes to owner
+      address a = msg.sender;
+      assembly {
+        priority := extcodehash(a)
+      }
+      require(bytes20(out.owner) == ripemd160(abi.encode(priority)), "Only UTXO owner or contract can start exit");
```

Comment:
Note: this is the keccak256 hash of the bytecode. (What we want) but not reflected on our other codebase/ tools yet? 🤔 

Also, should check for `zero`, in case the contract doesn't exist